lib/capybara/playwright/driver_extension.rb

```diff
@@ -40,6 +40,33 @@ def with_playwright_page(&block)
 
         @browser&.with_playwright_page(&block)
       end
+
+      # Start Playwright tracing (doc: https://playwright.dev/docs/api/class-tracing#tracing-start)
+      def start_tracing(name: nil, screenshots: false, snapshots: false, sources: false, title: nil)
+        # Ensure playwright page is initialized.
+        browser
+
+        with_playwright_page do |playwright_page|
+          playwright_page.context.tracing.start(name: name, screenshots: screenshots, snapshots: snapshots, sources: sources, title: title)
+        end
+      end
+
+      # Stop Playwright tracing (doc: https://playwright.dev/docs/api/class-tracing#tracing-stop)
+      def stop_tracing(path: nil)
+        with_playwright_page do |playwright_page|
+          playwright_page.context.tracing.stop(path: path)
+        end
+      end
+
+      # Trace execution of the given block. The tracing is automatically stopped when the block is finished.
+      def trace(name: nil, screenshots: false, snapshots: false, sources: false, title: nil, path: nil, &block)
```

Comment:
@yuki24 I prefer the method name `with_tracing` for this. Do you have any idea or opinion about this?

Reply:
Thank you for your kind advices.
I didn't know well about the Rails existing method names and its rules, and your advices really helped me.

Let's keep the method name 'with_trace' here.